app/lib/features/tasks/pages/my_tasks_page.dart

```diff
@@ -0,0 +1,150 @@
+import 'package:acter/features/home/providers/task_providers.dart';
+import 'package:acter/features/tasks/actions/my_task_actions.dart';
+import 'package:acter/features/tasks/widgets/task_item.dart';
+import 'package:acter_flutter_sdk/acter_flutter_sdk_ffi.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_easyloading/flutter_easyloading.dart';
+import 'package:acter/l10n/generated/l10n.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+
+class MyTasksPage extends ConsumerWidget {
+  const MyTasksPage({super.key});
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    final lang = L10n.of(context);
+    final sortedTasks = ref.watch(sortedTasksProvider).value;
+    ref.watch(myOpenTasksProvider);
```

Comment:
```suggestion
```

unnecessary, `myOpenTasksProvider` is already listened to in the sortedTaskProvider.